indico/client/request.py

```diff
@@ -48,6 +49,17 @@ def process_response(self, response):
 
 class RequestChain:
     previous: Any = None
+    
+    def __init__(self):
+        self.timeout = 0
 
     def requests(self):
         pass
+
+    def wait_request(self):
+        self.timeout += 0.5
```

Comment:
this needs a max. Also instead of putting this in request chain, you could just make a  Debouncer class so you don't need to worry about resetting it, you can just use one per call you want to do this in.

Reply:
Used a separate debouncer class.